pkg/sql/plan/function/list_builtIn.go

```diff
@@ -1272,6 +1272,35 @@ var supportedStringBuiltIns = []FuncNew{
 		},
 	},
 
+	// function `serial_extract`
+	{
+		functionId: SERIAL_EXTRACT,
+		class:      plan.Function_STRICT | plan.Function_ZONEMAPPABLE,
+		//TODO: verify with @m-schen
+		//layout:     STANDARD_FUNCTION,
+		layout: CAST_EXPRESSION,
```

Comment:
@m-schen Is it ok to use CAST_EXPRESSION here? Or should I use STANDARD_FUNCTION?

Reply:
the layout field just effect how the function shows in the layout of `explain`.

standard_function is func(arg0, arg1, arg2...)

and the cast is cast(arg0 as type).
I think it must not meet your requirement. you can write a new one for you if the standard is not what you want. @arjunsk

Reply:
Thank you for the suggestion. I will use `STANDARD_FUNCTION`.